shell.nix

```diff
@@ -0,0 +1,24 @@
+{ pkgs ? import <nixpkgs> {}
+, buildNodeJs ? pkgs.callPackage <nixpkgs/pkgs/development/web/nodejs/nodejs.nix> { python = pkgs.python3; }
+}: let
+  nodejsVersion = pkgs.lib.fileContents ./.nvmrc;
```

Comment:
if i don't use nvm, and i use nodenv, will this still work?

Reply:
nodevn will by default not respect the .nvmrc, but will with a separate plugin https://github.com/ouchxp/nodenv-nvmrc, though both nvm and fnm support it